ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -157,6 +157,7 @@ param set-default CBRK_SUPPLY_CHK 894281
 
 # disable check, no CPU load reported on posix yet
 param set-default COM_CPU_MAX -1
+param set-default COM_RAM_MAX -1
```

Comment:
Probably won't matter, but if needed we could customize this per architecture here. https://github.com/PX4/PX4-Autopilot/blob/1ae96d6509e000c05b05bf609709116fba9bc25f/platforms/nuttx/init/stm32h7/rc.board_arch_defaults